.github/workflows/python.yml

```diff
@@ -206,14 +206,6 @@ jobs:
         env:
           AWS_ACCESS_KEY_ID: ACCESSKEY
           AWS_SECRET_ACCESS_KEY: SECRETKEY
-    env:
-      AWS_ACCESS_KEY_ID: ACCESSKEY
-      AWS_SECRET_ACCESS_KEY: SECRETKEY
-      AWS_REGION: us-west-2
-      # this one is for s3
-      AWS_ENDPOINT: http://localhost:9000
-      # this one is for dynamodb
-      DYNAMODB_ENDPOINT: http://localhost:8000
```

Comment:
These are now provided inside of the tests. That way, we can run the tests with environment variables or with function arguments.